### ISSUE TITLE
image sequences

### ISSUE THREAD

**Issue by SFM2020** [BoomStarcuc/3DSfMFaceReconstruction#3]
How to input multi-view images when infering?  It looks like we can only input a single view image to get a disparity map in the inference.py

**Comment by baba-yaga** [comment]
Yes, I have the same question. Even when a directory containing an image sequence is fed into the program, it treats each image as a separate one. It would be nice to update the landmark positions accordingly. I am not sure that averaging would be best to do: say, the nose form is best revealed in the side view, so more weight should be given to the landmarks on the boundary of the face. I could not yet find how to obtain  the estimated head rotation information from fitting. Maybe the author could help us here?

**Comment by Leung-GD** [comment]
Did you slove this problem?

**Comment by sergei9838** [comment]
Hi, I have not been working on it. I find this project abandoned since the author provides no feedback and the current code just not working for me.
